### DISCUSSION TITLE
zh/guide/

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#772] [Announcements]
# zh/guide/

功能亮点 Bar ... Foo ... 

http://localhost:8080/zh/guide/

**Comment by song-code-rgd** [comment]
123

**Comment by liyuelian** [comment]
456

**Comment by song-code-rgd** [comment]
这是来自QQ邮箱的假期自动回复邮件。你好！我是宋立昌，有事我会稍后联系你，如果有急事可以联系我的QQ：2452221057,。
